cmd/storer2elastic/main.go

```diff
@@ -308,7 +308,7 @@ func startStorer2Elastic(ctx *cli.Context) error {
 			RatingsProcessor:          ratingsProcessor,
 			RatingConfig:              ratingsConfig,
 			StartingEpoch:             uint32(flagsValues.startingEpoch),
-			WaitingListFixEnableEpoch: nodeConfig.GeneralSettings.WaitingListFixEnableEpoch,
+			WaitingListFixEnableEpoch: 0, //TODO(next PR): remove the storer2elastic app
```

Comment:
add a JIRA task for this

Reply:
will be done automatically, without Jira task :)

Reply:
https://github.com/ElrondNetwork/elrond-go/pull/3121